README.md

```diff
@@ -42,8 +59,12 @@ also required to build this token.
 (starting with Crypto Express 4S adapters) configured with Enterprise PKCS#11
 (EP11) firmware.
 
+
+
 ## BUILD PROCESS
 
+**Note:** Building opencryptoki is only supported on AIX 7.2 and above. Attempts to build on older AIX releases will fail due to missing APIs.
```

Comment:
Maybe a slight rephrase:

Building opencryptoki **on AIX** is only supported on AIX 7.2....

Otherwise one could read it that way that opencryptoki can only be built on AIX (and not on a
other architectures).